spec/abstractops.html

```diff
@@ -1050,15 +1059,14 @@ <h1>ISO 8601 grammar</h1>
 
       TemporalTimeZoneString :
           TemporalTimeZoneIdentifier
-          TemporalInstantString
+          Date TimeSpecSeparator? TimeZone Calendar?
```

Comment:
Is the calendar annotation always required to go after the TZ annotation?  I assume yes, but wanted to make sure that there's not any expectation (e.g. on the IETF SEDATE list) that the order can be reversed. @ryzokuken?

Reply:
Yes, the grammar in the SEDATE draft also has this, and nobody has objected to it so far. All annotations can be in any order except TZ, which must either come first or not at all.